### ISSUE TITLE
Improve the Diary section of the app. 

### ISSUE THREAD

**Issue by HarryHeights** [ramack/ActivityDiary#262]
The activity diary is amazing. I've been using it since the beginning of this year and it has been helpful. 
However, I had some issues with the diary. 

There are two issues to be precise. 
- Inability to delete an activity registered in the diary.
- How the dairy is being displayed. 

### Inability to delete an activity registered in the diary.
The reason why this is an issue for me is because, starting an activity on the app is quite loose. By loose I mean too easy to execute. Often times I make the mistake of starting a wrong activity and once I stop that activity, it registers on the diary. 
If on the diary I'm able to delete this unwanted activity, it wouldn't have been a problem. 

#### My suggestion.
To solve the problem of unwanted activity, it's either users have to be extremely careful when starting an activity, so they don't start the wrong one. Or, another way is by initiating a confirmation protocol. That is, when a user wants to start an activity, he has to confirm if that's the activity he wants to start. 
Again, even with that protocol initiated, some people might still end up starting the wrong activity. 
That's why in my opinion, it will be great if activities can be deleted off the diary. 

### How the dairy is being displayed. 
Another issue I had with the diary was how it was being displayed. Every activity you perform in a day within a time frame is registered in the diary. 
The problem is how these activities are displayed in the diary. It's in a list format. What this means is that, if I want to check out an activity I did probably several months ago, I will have to engage in intense scrolling, which I didn't quite fancy. Although there's a search tool, I don't think it solves the problem completely. 

#### My suggestion. 

My suggestion would be to replace the current diary view to a calendar. Now with a calendar, dates would be easier to sort. I can easily go back and forth in time to view all the activities I have performed. 
If the diary is in a calendar format, I can just pick a previous date and check out the activities I performed that day, that is if I tracked them that day. 

Please check out my opinions and tell me what you think. Thanks.

**Comment by andswitch** [comment]
I agree with @HarryHeights first issue. It would be great if I am able to delete activities.
